test/e2e/framework/metrics_util.go

```diff
@@ -233,60 +232,21 @@ func (l *SchedulingMetrics) PrintJSON() string {
 	return PrettyPrintJSON(l)
 }
 
-type histogram struct {
-	Count   int
-	Buckets map[float64]int
+type Histogram struct {
+	Count   int            `json:"count"`
+	Buckets map[string]int `json:"buckets"`
```

Comment:
What are going to be values of the key? can you give some e.g?

Reply:
> New etcd metrics look: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/66158/pull-kubernetes-e2e-gce-100-performance/12890/artifacts/EtcdMetrics_density_2018-07-13T12:12:30Z.json

Those will be the strings used in original histograms. Each bucket label represents time upper bound, value of bucket represents number of  of operation with duration being less or equal given upper bound.

I'm not converting those strings to floats, cause json is unhappy with not-string labels.